src/nidmm/system_tests/test_system_nidmm.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/python
 
 import nidmm
+device_name = "dev1"
```

Comment:
Can you leave this the way it was? So it matches the name that a device would come up as (except DMM which is probably what you have?)

Or at least should be capitalized for correctness.